### PR TITLE
feat(dcs): add sdk for DCS instances and tags

### PR DESCRIPTION
**Pull request by chengxiangdong**
Add sdk for DCS instances and tags

### CODE REVIEW

File: openstack/dcs/v2/instances/requests.go [hunk 1]
```diff
@@ -0,0 +1,186 @@
+package instances
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/dcs/v2/tags"
```
Comment (ShiChangkuo) [inline openstack/dcs/v2/instances/requests.go:5]: can `github.com/chnsz/golangsdk/common/tags` meet your needs？

Comment (chengxiangdong) [reply openstack/dcs/v2/instances/requests.go:5]: The `value` in the API cannot be omitted, so the common tag cannot be used in DCS.

File: openstack/dcs/v2/tags/requests.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	})
+	if err == nil {
+		var r []ResourceTag
+		rst.ExtractInto(&r)
+		return r, nil
+	}
+	return nil, err
+}
+
+func List(c *golangsdk.ServiceClient, instanceID string) (*ResourceTagList, error) {
```
Comment (ShiChangkuo) [inline openstack/dcs/v2/tags/requests.go:68]: It seems that Get and List use the same URI, what's the difference between them?

File: openstack/dcs/v2/tags/requests.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+//ResourceTag is in key-value format
+type ResourceTag struct {
+	Key string `json:"key" required:"ture"`
+	// Value is not required, but it must be included in the parameter.
+	Value string `json:"value"`
+}
+
+type createTagsOpts struct {
```
Comment (ShiChangkuo) [inline openstack/dcs/v2/tags/requests.go:16]: suggest to rename this struct as it was used by Create and **Delete** methods.

Comment (chengxiangdong) [reply openstack/dcs/v2/tags/requests.go]: yes, good idea

File: openstack/dcs/v2/tags/requests.go (hunk 2 above)
Comment (chengxiangdong) [reply openstack/dcs/v2/tags/requests.go]: the List method should use listURL, I have modified it

**Pull request merged**